### ISSUE TITLE
Custom theme support removed in Files 2.4.21

### ISSUE THREAD

**Issue by lbux** [catppuccin/windows-files#7]
The option to add custom themes through files was removed as noted in several GitHub issues:

https://github.com/files-community/Files/issues/10315
https://github.com/files-community/Files/issues/10542
https://github.com/files-community/Files/issues/10885

The [updated documentation](https://files.community/docs/configuring/custom-themes) suggests some ways to customize it to an extent. Is there any way to integrate catppuccin to this new version?

**Comment by CaidenBlock** [comment]
According to the docs, the only thing you can do is change are background colors, all accents are based off of windows theme color.

**Comment by charliie-dev** [comment]
Sad to see this. 😢

**Comment by Only-Moon** [comment]
Shouldn't this repo be archived since it can be applied anymore?

**Comment by Only-Moon** [comment]
Shouldn't this repo be archived since it can be applied anymore?

**Comment by Aworldc** [comment]
Got it working!

```json
{
  "AppThemeBackgroundColor": "#ff232634",
  "AppThemeAddressBarBackgroundColor": "#ff292c3c",
  "AppThemeSidebarBackgroundColor": "#ff292c3c",
  "AppThemeFileAreaBackgroundColor": "#ff303446"
}
```

<img width="1280" alt="image" src="https://github.com/catppuccin/windows-files/assets/83331712/692249fa-88c9-4ceb-99e4-670d862304eb">

**Comment by thiphariel** [comment]
> Got it working!
> 
> ```json
> {
>   "AppThemeBackgroundColor": "#ff232634",
>   "AppThemeAddressBarBackgroundColor": "#ff292c3c",
>   "AppThemeSidebarBackgroundColor": "#ff292c3c",
>   "AppThemeFileAreaBackgroundColor": "#ff303446"
> }
> ```
> 
> <img alt="image" width="1280" src="https://private-user-images.githubusercontent.com/83331712/280430673-692249fa-88c9-4ceb-99e4-670d862304eb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI4OTM2MTMsIm5iZiI6MTcwMjg5MzMxMywicGF0aCI6Ii84MzMzMTcxMi8yODA0MzA2NzMtNjkyMjQ5ZmEtODhjOS00Y2ViLTk5ZTQtNjcwZDg2MjMwNGViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE4VDA5NTUxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcxY2Q5M2JlMjllZGQ5NjcwMmZiMTZlZjRjMjJhMGI2ZjllNTMwZjFmOTkyZDlhZWNmZWFiNmU0Yjg2MTZjODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FLlm-ED0IdSbM9byf04CNPF7T6xCKkjAST0jq8_YqTc">

@Aworldc ~~Hi! How did you manage to do it? My settings file got erased (manual changes) after every restart... Thanks :-)~~

Ok well, I just had to disable the background running task... Sorry for that! And thanks for the theme tip :-) Can confirm it still "works"

**Comment by yair100** [comment]
> According to the docs, the only thing you can do is change are background colors, all accents are based off of windows theme color.

@CaidenBlock the current options are based on the brushes that users were customizing in the previous theming system, but we can add more options if there is enough interest.

**Comment by DakshG07** [comment]
@yaira2 it would be quite nice if additional options were available. The theme’s 80% there, but it would be great if we could really theme everything. Was there a specific reason to limit theme functionality? In my opinion, the more customization options, the better.

Also, I’ll test out the new theme methods and update this repository sometime today or tomorrow.

**Comment by yair100** [comment]
The previous system worked by loading xaml into the app at startup and overriding the default brushes in Files. This was a good idea in theory, but in actuality we found it had an impact on startup performance, the syntax was too complex for most users, and was prone to breaking whenever we released updates to Files. On the flip side, the current theming system uses json, has good performance, and doesn't have any issues when the app is updated.
